### DISCUSSION TITLE
I want to develop a game called God of nutrients<title>

### DISCUSSION THREAD

**Discussion by DavidGhost74** [community/community#46431] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I'm new here, but I don't know how to code.i have alot of game ideas but I can't code .I need help to develop my games. Or I will give the ideas out  and we will discuss about it

**Comment by bestegipster** [comment]
Hey there! I think God of Nutrients sounds like an interesting game idea. If you're not familiar with coding, you could try using game engines like Unity or GameMaker Studio. They have user-friendly interfaces and can help you turn your ideas into a working prototype. You could also look into hiring a freelance developer to help bring your vision to life. As for inspiration, you could check out some fitness apps or games that already exist and see what you like or don't like about them. For instance, I recently [discovered the blackout bingo review](https://joywallet.com/article/blackout-bingo-review/), and this review helped me with my own project.
